AdminApi/src/AdminApi/ClientApp/src/app/app-routing.module.ts

```diff
@@ -23,6 +24,7 @@ const routes: Routes = [
     { path: "clients", component: ClientListComponent, data: { breadcrumb: "Clients" }, canActivate: [AuthGuard] },
     { path: "clients/create", component: ClientEditComponent, data: { breadcrumb: "Create Client" }, canActivate: [AuthGuard] },
     { path: "clients/:id", component: ClientEditComponent, canActivate: [AuthGuard] },
+    { path: "deletion-process-details/:address/:id", component: DeletionProcessDetailsComponent, canActivate: [AuthGuard] },
```

Comment:
It would be a good idea to add it not to display double ID-s for no reason. I suggest that we name that breadcrumb data "Identity Deletion Process"? If you had anything else in mind please say so :)